pongObjects_Methods.js

```diff
@@ -0,0 +1,157 @@
+// get canvas properties 
+const cvs = document.getElementById("PongTable");
```

Comment:
this is vanilla JS so you should use var for backwards compatibility on older browsers

Reply:
var pongTable = $('#PongTable')